default.nix

```diff
@@ -22,8 +22,14 @@ let
   # nix-index takes a little while to run and the contents don't change
   # meaningfully very often.
   indexCache = fetchurl {
-    url = "https://github.com/Mic92/nix-index-database/releases/download/2021-12-12/index-x86_64-linux";
-    sha256 = "sha256-+SoG5Qz2KWA/nIWXE6SLpdi8MDqTs8LY90fGZxGKOiA=";
+    url =
+      if stdenv.isDarwin
+      then "https://github.com/Mic92/nix-index-database/releases/download/2022-02-27/index-x86_64-darwin"
+      else "https://github.com/Mic92/nix-index-database/releases/download/2021-12-12/index-x86_64-linux";
+    hash =
+      if stdenv.isDarwin
+      then "sha256-sHGUSjd6EOpzdWtS5FGtTkS9KEKvDCGMHTYVwxOkZIo="
+      else "sha256-+SoG5Qz2KWA/nIWXE6SLpdi8MDqTs8LY90fGZxGKOiA=";
```

Comment:
There might be more platforms in the future. Please use attr select instead.

Reply:
Ähm sorry, what do you mean with `attr` select (haven't found something obvious in the internet).

Reply:
something like this https://github.com/NixOS/nixpkgs/blob/980ed8dd768d13dd020a4a7f96a9b55882bdae19/pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix#L94-L116

Reply:
Solved, thanks for the feedback.